runtime/lib/backend/common/CacheManager.hpp

```diff
@@ -58,15 +62,18 @@ class CacheManager {
      */
     void Reset()
     {
-        this->ops_names_.clear();
```

Comment:
I must admit I liked these explicit `this`s in the code: I think that they prevented some possible ambiguity when reading. But its fine if you really want to remove them.

Reply:
yeah I agree that it generally reduces ambiguity and helps the code readability. But, in the PL C++ codebase (Lightning sims & Catalyst), we agreed upon using post-fixed `_` to indicate private class variables and reduce the use of `this` in the codebase. In Catalyst, we still use `this` in some classes and follow other styles for private variables, but I'd like to see consistency in the codebase with [this new style](https://github.com/PennyLaneAI/dev_guidelines/tree/main/cpp) in the long run.